notebooks/Hugging_Face_VisionTranformer.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<p>Do you think it makes sense to explain why this is not wrapped together in ViT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/skorch-dev/skorch/pull/917/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Sorry, I don't understand. What is not wrapped together?

Reply:
as in, `FeatureExtractor` not being chained with the `ViTModel` inside `VitModule` and being separately put.

Reply:
Yes, makes sense, I added an explanation.